pkg/storage/pluginstore/store.go

```diff
@@ -61,11 +61,13 @@ func (s *Store) Connect() error {
 	}
 
 	s.BackingStore = crud.NewBackingStore(store)
+	s.BackingStore.AutoClose = false
 
 	return nil
 }
 
 func (s *Store) Close() error {
+	return nil
```

Comment:
Was this line intentional?

Reply:
Nope! Thanks, that's weird not sure where that came from.